src/components/loadable.cjsx

```diff
@@ -21,14 +23,17 @@ module.exports = React.createClass
     renderError: React.PropTypes.func.isRequired
     renderBug: React.PropTypes.func.isRequired
 
+  getInitialState: ->
+    refreshButton: false
+
   getDefaultProps: ->
 
     # Enables a renderStatus prop function with a component other than a div
-    renderLoading: ->
-      <div className='-loading'>Loading...</div>
+    renderLoading: (refreshButton) ->
```

Comment:
I think if it always prepends, that would work fine and I can style the refresh button on the calendar loading screen as needed for now.

It would be nice if the button remained an argument that is a component for the custom render functions to use within the screens, but we can update that later.